### ISSUE TITLE
Review the PULFALight Solr schema for field indexing

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/pulfalight#145]
There may need to be customization of the Solr schema for PULFALight installation given that certain fields may need to be adjusted.

It also may be the case that the upstream ArcLight Solr configuration is perfectly acceptable.

**Comment by sdellis** [comment]
@jrgriffiniii I'm not sure if this comment should go here, but before we can make any requests to Aeon, all components need to have complete and accurate container information. How can we facilitate this?

**Comment by jrgriffiniii** [comment]
I definitely agree, this needs to be stated. I had hoped to use this issue in order to review the configuration for Solr fields in the Collection, and hence, I can create a separate issue addressing this.

**Comment by tpendragon** [comment]
Our entire indexing strategy is customized now, with our own tests. Closing this.
